core/misc/autocomplete.js

```diff
@@ -145,7 +145,7 @@ Backdrop.jsAC.prototype.selectUp = function () {
  * Highlights a suggestion.
  */
 Backdrop.jsAC.prototype.highlight = function (node) {
-  // Unhighlights a suggestion for "keyup" and "keydown" events.
+  // Un-highlights a suggestion for "keyup" and "keydown" events.
```

Comment:
Legit: https://en.wiktionary.org/wiki/unhighligh

Reply:
There is [*unhighlighted*](https://www.merriam-webster.com/dictionary/unhighlight), but not *unhighlight*.